docs/developer_guide/lane_pose_calculation/LanePoseCalculation.md

```diff
@@ -0,0 +1,83 @@
+# Lane pose calculation
+
+The calculation method of the lane coordinate system differs from Entity to Entity, and the process of determining which lane to match is complex, including fallback.  
+In this document, we will show how the pose calculation of the lane coordinate system is performed for each Entity.
+
+There are 3 possible timings when the lane coordinate system may be computed for all Entities.  
+These are the [timing immediately after the spawn of the Entity](Spawn.md), the [timing of the frame update](UpdateFrame.md) the [timing of the getting longitudinal distance](GetLongitudinalDistance.md)
+
+!!! Note
+    If you are not a simulator developer but a scenario writer, it is sufficient to check [this document](UpdateFrame.md) and [this section](#lane-coordinate-system-calculation-algorithm-for-a-specific-lane), since the distance measurement via condition in used in the OpenSCENARIO is basically done after frame update.
```

Comment:
```suggestion
    It is sufficient to check [this document](UpdateFrame.md) and [this section](#lane-coordinate-system-calculation-algorithm-for-a-specific-lane), if you are not a simulator developer but a scenario writter. That is because the distance measurement via condition of the OpenSCENARIO is done after frame update basically.
```